bgpd/bgpd.c

```diff
@@ -3051,6 +3053,10 @@ int bgp_get(struct bgp **bgp_val, as_t *as, const char *name,
 	}
 
 	bgp = bgp_create(as, name, inst_type);
+	if (bgp_option_check(BGP_OPT_EMULATE_VRF)) {
+		if (name)
+			bgp->vrf_id = vrf_generate_id();
```

Comment:
I think we can combine the two `if` statements above into a single one for simplicity.

Reply:
ok